web-api/src/business/useCases/processStreamRecords/processCaseCorrespondenceEntries.ts

```diff
@@ -1,5 +1,5 @@
 import { RawCorrespondence } from '@shared/business/entities/Correspondence';
-import { getLogger } from 'aws-xray-sdk';
+import { getLogger } from '@web-api/utilities/logger/getLogger';
```

Comment:
Thank you for catching this! This is on me, but I also think it might be wise to rename our logger.